src/tools/MacOS Installation.md

```diff
@@ -0,0 +1,47 @@
+# MacOS Installation
+
+### 1. Download
```

Comment:
These steps should start with a secondary header, perhaps something like `## Installation steps`?

Reply:
Fixed this for now